test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2018 The Kubernetes Authors.
-
```

Comment:
why these changes? they break `verify` tests.

Reply:
Fixed.. not sure why this got changed. Thanks